### PR TITLE
[New Term] C++ Math Functions: `rint()`

### LINKED ISSUE #985: [Term Entry] C++ Math Functions: `rint()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `rint()` function.

- In the terms/ folder, make a new folder called `rint`
- Inside the `rint` folder, create a new markdown file called `rint.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by chiragcode11**
Description
Closes #985 

Added rint() function within the C++ arithmetic library.
Added description with examples.
Type of Change
Added rint() function within the C++ Math library.
Checklist
 

- [x] All writings are my own.
- [x]  My entry follows the Codecademy Docs style guide.
- [x]  My changes generate no new warnings.
- [x]  I have performed a self-review of my own writing and code.
- [x]  I have checked my entry and corrected any misspellings.
- [x]  I have made corresponding changes to the documentation if needed.
- [x]  I have confirmed my changes are not being pushed from my forked main branch.
- [x]  I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x]  Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes # in the "Description" above).

### CODE REVIEW

**Comment by chiragcode11** [comment]
I have created a new PR from another branch

**Comment by Dusch4593** [comment]
Thank you very much @chiragcode11 😄 Taking a look at this now.

File: content/cpp/concepts/math-functions/terms/rint/rint.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/rint/rint.md:14]:
```suggestion

The **`rint()`** function rounds the argument to an integral value using the current rounding mode.

```

File: content/cpp/concepts/math-functions/terms/rint/rint.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+## Syntax
+
+```pseudo
+rint(x)
+```
+
+Argument must be a `double`/`float`/`long double`, and the return value will be same type.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/rint/rint.md:21]:
```suggestion
The `x` parameter must be a `double`, `float`, or `long double`. The return value will be the same data type.
```

File: content/cpp/concepts/math-functions/terms/rint/rint.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Return value
+
+The rint() function rounds the argument x to an integral value, using the rounding direction specified by fegetround() and returns the value. By default, the rounding direction is set to 'to-nearest'. The rounding direction can be set to other values using fesetround() function.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/rint/rint.md:25]:
```suggestion

The rounding direction specified by the `fegetround()` function, with the default direction set as 'to-nearest'. The rounding direction can be set to other values using `fesetround()` function.
```

File: content/cpp/concepts/math-functions/terms/rint/rint.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## Example
+
+The following example uses the `rint()` function to find the value of x after rounding off:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/rint/rint.md:30]:
```suggestion
The following example uses the `rint()` function to find the value of `x` after rounding off:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@chiragcode11 Finished with my review and left you some comments/suggestions to implement. 😄

**Comment by chiragcode11** [comment]
updated the suggestion you gave, thank you.

File: content/cpp/concepts/math-functions/terms/rint/rint.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Syntax
+
+```pseudo
+rint(x)
+```
+
+The `x` parameter must be a `double`, `float`, or `long double`. The return value will be the same data type.
+
+The rounding direction specified by the `fegetround()` function, with the default direction set as 'to-nearest'. The rounding direction can be set to other values using `fesetround()` function.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/rint/rint.md:25]:
```suggestion
The rounding direction is specified by the `fegetround()` function, with the default direction set as 'to-nearest'. The rounding direction can be set to other values using `fesetround()` function.
```

**Review by Dusch4593 — APPROVED** [review]
@chiragcode11 LGTM for the next review! 😄

Just left one minor suggestion.

**Comment by chiragcode11** [comment]
@Dusch4593 done as per your suggestion

**Comment by chiragcode11** [comment]
can we get a hacktoberfest verified label

**Comment by Dusch4593** [comment]
@chiragcode11 Since I conducted the 1st review, @HishamT will conduct the second. 😄

**Review by SSwiniarski — APPROVED** [review]
@chiragcode11, @Dusch4593 Dropped by for a second review, and everything looks good as-is.

**Review by Dusch4593 — APPROVED** [review]
@chiragcode11 I just had one more suggestion. 😄

File: content/cpp/concepts/math-functions/terms/rint/rint.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+---
+Title: 'rint()'
+Description: 'The rint() function in C++ rounds the argument to an integral value using the current rounding mode.'
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/rint/rint.md:3]:
```suggestion
Description: 'Rounds the argument to an integral value using the current rounding mode.'
```

**Comment by Dusch4593** [comment]
@chiragcode11 This looks ready to merge! I made a few last-minute edits in bdf0589105c3a67be6b34123f9de4976c295b2c2, including the following:

- Removed extra newline.
- Converted text about `fegetround()`/`fesetround()` to a **note**.

**Comment by Dusch4593** [comment]
Congrats, @chiragcode11! Your contribution to Codecademy Docs should now be live! It can be viewed here: https://www.codecademy.com/resources/docs/cpp/math-functions/rint

You're currently anonymous, but if you want to make yourself public, make sure to add your GitHub handle to your Codecademy profile:

Screen Shot 2022-10-25 at 5 16 25 PM

**Pull request merged**